score_test.go

```diff
@@ -1,51 +0,0 @@
-package zoekt
-
-import (
-	"maps"
-	"testing"
-)
-
-func TestCalculateTermFrequency(t *testing.T) {
```

Comment:
Now `calculateTermFrequency` requires access to `indexData`, so it's hard to unit test. I checked the e2e test `scoring_test.go` carefully to confirm the new calculation is correct.